src/jit/compiler.h

```diff
@@ -1771,7 +1773,11 @@ class Compiler
     // a PSPSym for functions with any EH.
     bool ehNeedsPSPSym() const
     {
+#ifdef _TARGET_X86_
```

Comment:
@janvorli According to https://github.com/dotnet/coreclr/blob/master/Documentation/botr/clr-abi.md#x86-eh-considerations, all the variables are accessed via frame pointer (EBP) and the managed code always stores frame pointer in its frame. That is one reason that I disable this.

In addition, I encountered various assertions failures that I could't understand when I tried to enable PSPSym-related code.